apps/collection/js/setup.js

```diff
@@ -1,11 +1,9 @@
 'use strict';
 
-/* global CollectionsDatabase */
+/* global CollectionsDatabase, LazyLoader */
 
 (function(exports) {
 
-  const PRE_INSTALLED_COLLECTIONS_FILE = 'js/pre_installed_collections.json';
```

Comment:
Do not remove this declaration, use it instead in the code below like the existing code does.